orb-ui/src/engine/pearl.rs

```diff
@@ -603,6 +603,18 @@ impl EventHandler for Runner<PEARL_RING_LED_COUNT, PEARL_CENTER_LED_COUNT> {
                             sound::Voice::VerificationNotSuccessfulPleaseTryAgain,
                         ))?;
                     }
+                    SignupFailReason::SoftwareVersionDeprecated => {
+                        self.operator_blink.trigger(
+                            Argb::PEARL_OPERATOR_VERSIONS_DEPRECATED,
+                            vec![0.4, 0.4, 0.4, 0.4, 0.4, 0.4],
```

Comment:
```suggestion
                            vec![0.4; 6],
```

Reply:
it's more rusty but less easy visually to count the number of blinks 😁